calendar/src/main/java/org/exoplatform/platform/qa/ui/calendar/pageobject/CalendarHomePage.java

```diff
@@ -129,10 +127,14 @@ public void goToRightMenuTaskEventFromWeekView(String name, selectDayOption opti
     if (date != null && date != "") {
       switch (optionDay) {
         case DETAILTIME:
+          sleep(3000);
```

Comment:
Avoid using sleep please.

Reply:
Done Fixed.

Reply:
The remark should be applied everywhere.

Reply:
Yes done.